lib/s3middleware/tagging.ts

```diff
@@ -32,6 +37,12 @@ export const _validator = {
         && tag.Key[0] !== undefined && tag.Value[0] !== undefined
         && typeof tag.Key[0] === 'string' && typeof tag.Value[0] === 'string',
 
+    validateTagObjectStructure: (tag: BucketTag) => tag
+        && Object.keys(tag).length === 2
+        && tag.Key && tag.Value
+        && tag.Key.length >= 1 && tag.Value.length >= 1
+        && typeof tag.Key === 'string' && typeof tag.Value === 'string',
```

Comment:
I would switch these two comparisons, if the tag key or value is not a string it is not always going to have a `.length`, eg if either were numbers.

Reply:
Done: https://github.com/scality/Arsenal/pull/1933/commits/4b19cc2101f4a15666f8746ad8fb12533e9b45d8#diff-7aa7eeaed88643ea97d57a5f15ddc2e64c380190641c53bb0c3f64d67807a475R46